common/persistence/dataInterfaces.go

```diff
@@ -304,6 +318,22 @@ type (
 		EventID             int64
 	}
 
+	// ReplicationTask is the transfer task created for shipping history replication events to other clusters
+	ReplicationTask struct {
+		TaskID              int64
+		DomainID            string
```

Comment:
is this DomainID necessary?

Reply:
You are right.  Replication task is always going to be for the same domain.  Removed.